ChangeLog.d/ssl_erver_ticket_aead_option.txt

```diff
@@ -0,0 +1,4 @@
+Changes
```

Comment:
I'm not sure this needs a ChangeLog entry. It's mostly and internal testing enhancement. OTOH, perhaps some users are using those programs and would like to know about the new option?

Reply:
`ssl_*` are firmly in the test category. We don't provide any kind of backward compatibility for them. We shouldn't announce a feature in the changelog, since we could take it out without announcing it.